Gruntfile.js

```diff
@@ -51,13 +51,14 @@ module.exports = function(grunt) {
     }
   };
 
-  var prefixMatch = "\\n *(function |var |static )?";
+  var regexLF = (grunt.util.linefeed === "\r\n" ? "(\\u000D\\n)" : "\\n"); //get the correct Lf for the operating system
```

Comment:
Also, ```\\u000D``` represents a unicode Carriage Return in the regular expression. I played with various alternatives, this was the one that gave the result.